src/include/sof/trace.h

```diff
@@ -200,20 +200,8 @@ _TRACE_EVENT_NTH_DECLARE_GROUP(3)
  */
 _TRACE_EVENT_NTH_DECLARE_GROUP(4)
 
-/* Declaration of
- * void _trace_event5            (uint32_t log_entry, uint32_t ids...,
- *                                uint32_t params...);
- * void _trace_event_mbox5       (uint32_t log_entry, uint32_t ids...,
- *                                uint32_t params...);
- * void _trace_event_atomic5     (uint32_t log_entry, uint32_t ids...,
- *                                uint32_t params...);
- * void _trace_event_mbox_atomic5(uint32_t log_entry, uint32_t ids...,
- *                                uint32_t params...);
- */
-_TRACE_EVENT_NTH_DECLARE_GROUP(5)
+#define _TRACE_EVENT_MAX_ARGUMENT_COUNT 4
 #endif
-#define _TRACE_EVENT_MAX_ARGUMENT_COUNT 5
```

Comment:
Why move `_TRACE_EVENT_MAX_ARGUMENT_COUNT` inside the `#ifdef`?
Isn't the maximum number of supported arguments in `_trace_event` is same regardless of platform? (while the new implementation of `_trace_event` on `host` indeed allows indefinite - as long as stack allows - number of arguments).

Reply:
That's my bad, screwed up merging after rebase. Fixing now.